spec/rubocop/cop/rspec/contain_exactly_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::ContainExactly do
+  it 'flags `contain_exactly` with only splat arguments' do
+    expect_offense(<<-RUBY)
+      it { is_expected.to contain_exactly(*array1, *array2) }
+                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `match_array` when matching array values.
+    RUBY
+
+    expect_correction(<<-RUBY)
+      it { is_expected.to match_array(array1 + array2) }
+    RUBY
+  end
+
+  it 'flags `contain_exactly` with a splatted percent literal array' do
+    expect_offense(<<-RUBY)
+      it { is_expected.to contain_exactly(*%w(a b)) }
+                          ^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `match_array` when matching array values.
+    RUBY
+
+    expect_correction(<<-RUBY)
+      it { is_expected.to match_array(%w(a b)) }
+    RUBY
+  end
+
+  it 'ignores non-splat usages' do
+    expect_no_offenses(<<-RUBY)
+      it { is_expected.to contain_exactly([1,2,3]) }
+      it { is_expected.to contain_exactly(a.merge(b)) }
+      it { is_expected.to contain_exactly((a + b)) }
```

Comment:
All those examples were meant to be splats

Reply:
```diff
-       it { is_expected.to contain_exactly(*[1,2,3]) }
-       it { is_expected.to contain_exactly(*a.merge(b)) }
-       it { is_expected.to contain_exactly(*(a + b)) }
+       it { is_expected.to match_array([1,2,3]) }
+       it { is_expected.to match_array(a.merge(b)) }
+       it { is_expected.to match_array((a + b)) }
```
Does this look fine?